spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
+  as the `id` in the DIDDoc.
+- When a `did:webs` is permanently moved to some other location, an event in the
+  KEL records the change.
+  - The `id` in the DIDDoc is set to the new location.
+  - An `AlsoKnownAs` entry is added to the DIDDoc for the old location.
+  - If possible, the controller of the DID MAY use web redirects to allow
+    resolution of the old location of the DID to the new location.
+
+If the previously published location of a `did:webs` is not redirected, an
+entity trying to resolve the DID MAY be able to find the data for the DID
+somewhere else using just the AID. Since the AID is globally unique and
+references the same identifier, regardless of the rest of the string that is the
+full `did:webs`, web searching could yield either the current location of the
+DIDDoc, or a copy of the DID that may be useful. For example, even the [Internet
+Archive: Wayback Machine](https://archive.org/web) could be used to find a copy
+of the DIDDoc and KEL at some point in the past, that may be sufficient for the
+purposes of the entity trying to resolve the DID. This specification does not
+rely on the Wayback Machine, but it might be a useful DID resolver tool.
+
+The DIDDoc, KEL and other files related to the DID may be cop
+
+### Identifiers in a `did:webs` DIDDoc
+
+Within the DIDDoc of a `did:webs` DID, the following rules are followed:
+
+- The `id` field in the root of the DIDDoc is the `did:webs` DID, reflecting the current combined web location and AID of the identifier.
+- The `alsoKnownAs` field in the root of the DIDDoc MAY contain other synonym resolvable `did:webs` DIDs. The `alsoKnownAs` field MAY contain `did:web` versions of the `did:webs` DID(s).
+- `id` items elsewhere in the DIDDoc MUST use the relative form of the identifier, e.g., `"id" : "#<identifier>"`.
 
 ### Key Material and Document Handling
 
 DID docs for this method are pure JSON. They may be processed as JSON-LD by
 prepending an `@context` if consumers of the documents wish.
 
-DID URLs in DID docs MUST be absolute and MUST use the [[ref: stable]] form of the
-DID's MSI.
+The KELs are also pure JSON and are processed per the KERI specification rules.
 
 ### DID Method Operations
 
 #### Create (Register)
 
 Creating a DID involves these steps:
 
-Create a KERI AID. This process is documented in KERI materials.
-
-Choose a [[ref: DDURL]] where the DID doc for the DID will be published.
-
-Append `.kel` to the DDURL and publish the KEL of the AID at this location.
-
-Derive the DID doc from the KEL and publish it at the DDURL. TODO: how!
+- Choose the web URL where the DID doc for the DID will be published, excluding
+  the last element that will be the AID, once defined.
+- Create a KERI AID and add the appropriate KERI events that will (at least)
+  produce the DIDDoc. That process is described in the [DIDDoc and KEL](#)
+  generation section of this specification.
+- Choose the web URL where the DID doc for the DID will be published, including
```

Comment:
I don't understand the partial redundancy between this line and the one two items above.

Reply:
Updated to (hopefully) address the question.  There might be a chicken and egg with the KERI events to create the AID and the inclusion of the DID web location in the DIDDoc.  I suspect not, as all you need to produce is the inception event to get the AID and I’m assuming that the DID is not part of the inception event.

We can define this further if the chicken and egg is a non-issue. This gets into the KEL/TEL events that are needed in creating the DID and DIDDoc.